spec/rubocop/cop/rspec/capybara/specific_finders_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::SpecificFinders, :config do
+  it 'registers an offense when using `find`' do
+    expect_offense(<<~RUBY)
+      find('#some-id')
+      ^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      find_by_id('some-id')
+    RUBY
+  end
+
+  it 'registers an offense when using `find` and other args' do
+    expect_offense(<<~RUBY)
+      find('#some-id', exact_text: 'foo')
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      find_by_id('some-id', exact_text: 'foo')
+    RUBY
+  end
+
+  it 'registers an offense when using `find` with method chain' do
+    expect_offense(<<~RUBY)
+      find('#some-id').find('#other-id').find('#another-id')
+      ^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
+                       ^^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
+                                         ^^^^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      find_by_id('some-id').find_by_id('other-id').find_by_id('another-id')
+    RUBY
+  end
+
+  it 'does not register an offense when using `find` ' \
+    'with argument is not id' do
+    expect_no_offenses(<<~RUBY)
+      find('a.some-id')
+      find('.some-id')
+    RUBY
+  end
+
+  it 'does not register an offense when using `find_by_id`' do
+    expect_no_offenses(<<~RUBY)
+      find_by_id('some-id')
+    RUBY
+  end
+
+  it 'does not register an offense when using `find` ' \
+    'with argument is id with multiple matcher' do
+    expect_no_offenses(<<~RUBY)
+      find('#some-id body')
+      find('#some-id,h1')
+      find('#some-id,h2')
+      find('#some-id+option')
```

Comment:
what about the case of `find('[id=something]')`

Reply:
Also is it possible to write `find('h2#some')` ?

Reply:
https://github.com/rubocop/rubocop-rspec/blob/246e30e2c4cfdbb5762c0c6ec8475c356893273a/spec/rubocop/cop/rspec/capybara/specific_finders_spec.rb#L39-L48

Reply:
I have summarized them below. What do you think?

https://github.com/rubocop/rubocop-rspec/blob/246e30e2c4cfdbb5762c0c6ec8475c356893273a/spec/rubocop/cop/rspec/capybara/specific_finders_spec.rb#L39-L77